app/models/concerns/curator/metastreamable.rb

```diff
@@ -9,7 +9,7 @@ module All
         include Metastreams::Administratable
         include Metastreams::Workflowable
         include Metastreams::Descriptable
-        include Metastreamable::InstanceMethods
+        # include Metastreamable::InstanceMethods
```

Comment:
Should we remove these commented-out modules? Or leave, if we might need them later?

Reply:
Just pushed a commit removing those